librz/asm/arch/6502/6502dis.c

```diff
@@ -143,40 +143,49 @@ static int _6502Disass(ut64 pc, RzAsmOp *op, const ut8 *buf, ut64 len) {
 	int i;
 	for (i = 0; ops[i].name != NULL; i++) {
 		if (ops[i].op == buf[0]) {
-			const char *buf_asm = "invalid";
+			char *buf_asm = rz_str_newf("invalid");
```

Comment:
You could start as `NULL` and if it stays NULL before the end then you can set this as `rz_str_new("invalid")`